packages/core/package-manager/src/NodePackageManager.js

```diff
@@ -26,8 +27,12 @@ import {getModuleParts} from '@parcel/utils';
 import {getConflictingLocalDependencies} from './utils';
 import {installPackage} from './installPackage';
 import pkg from '../package.json';
-import {NodeResolver} from './NodeResolver';
-import {NodeResolverSync} from './NodeResolverSync';
+import {Resolver} from '@parcel/node-resolver-core/index';
```

Comment:
```suggestion
import {Resolver} from '@parcel/node-resolver-core';
```

Reply:
Yeah I need to expose the underlying Rust implementation rather than the wrapper from that package for this